pkg/tabulator/tabstate.go

```diff
@@ -169,8 +174,8 @@ func Update(ctx context.Context, client gcs.ConditionalClient, mets *Metrics, co
 			log.WithFields(logrus.Fields{
 				"from": fromPath.String(),
 				"to":   toPath.String(),
-			}).Info("Copying state")
-			if confirm {
+			}).Info("Calculating state")
+			if copyOnly && confirm {
```

Comment:
nit: If confirm = false, think neither copy nor upload should happen?

Reply:
Yes, that's right. There was a bug here my tests didn't catch; thanks!